locale/en/docs/guides/simple-profiling.md

```diff
@@ -86,10 +93,15 @@ high latency on requests. We can easily run the app with the built in profiler:
 NODE_ENV=production node --prof app.js
 ```
 
-and put some load on the server using `ab` (ApacheBench):
+We'll create a new user:
 
 ```
 curl -X GET "http://localhost:8080/newUser?username=matt&password=password"
+```
+
+and put some load on the server using `ab` (ApacheBench):
+
+```
 ab -k -c 20 -n 250 "http://localhost:8080/auth?username=matt&password=password"
```

Comment:
While we are here. It may be worthwhile to use [autocannon](https://npm.im/autocannon) with `npx` instead. Installing `ab` is much more involved.

Reply:
@kcacciatore thank you for your contribution. Feel free to the the action the above comment if you feel up for it, or leave this for us to do in a follow-on.

Reply:
@ofrobots - I'll leave it for now - I am totally new to node.js, and typically go learn to debug/profile first when I learn something new. I'm not familiar with autocannon or npx, but will look into it.

Reply:
You can basically achieve the same with:

```
npx autocannon -c 20 -a 250 "http://localhost:8080/auth?username=matt&password=password"
```

This has the benefit of not requiring anything complex to install (especially on Windows).